lib/src/messages/checkout_session.dart

```diff
@@ -52,15 +52,18 @@ class CheckoutSession extends Message {
   final String? customer;
   final String? paymentIntent;
   final List<PaymentMethodType> paymentMethodTypes;
+  final String? status;
+  final String? url;
 
-  CheckoutSession({
-    required this.object,
-    required this.id,
-    required this.paymentMethodTypes,
-    this.clientReferenceId,
-    this.customer,
-    this.paymentIntent,
-  });
+  CheckoutSession(
+      {required this.object,
+      required this.id,
+      required this.paymentMethodTypes,
+      this.clientReferenceId,
+      this.customer,
+      this.paymentIntent,
+      this.status,
+      this.url});
```

Comment:
Could you please add trailing comma to improve formatting?

Reply:
Done